diff --git a/node_modules/@comunica/bus-rdf-update-quads/lib/ActorRdfUpdateQuadsDestination.js b/node_modules/@comunica/bus-rdf-update-quads/lib/ActorRdfUpdateQuadsDestination.js
index 7583958..ac636b9 100644
--- a/node_modules/@comunica/bus-rdf-update-quads/lib/ActorRdfUpdateQuadsDestination.js
+++ b/node_modules/@comunica/bus-rdf-update-quads/lib/ActorRdfUpdateQuadsDestination.js
@@ -10,7 +10,8 @@ function deskolemizeStream(stream, id) {
 exports.deskolemizeStream = deskolemizeStream;
 function deskolemize(action) {
     const destination = action.context.get(context_entries_1.KeysRdfUpdateQuads.destination);
-    const id = action.context.get(context_entries_1.KeysRdfResolveQuadPattern.sourceIds)?.get(destination);
+    // const id = action.context.get(context_entries_1.KeysRdfResolveQuadPattern.sourceIds)?.get(destination);
+    const id = '0'
     if (!id) {
         return action;
     }
